template/web/wp-content/themes/<%= slug %>/app/Managers/TwigManager.php

```diff
@@ -16,12 +17,38 @@ class TwigManager implements ManagerInterface {
 	 */
 	public function run() {
 		add_filter( 'timber/twig', array( $this, 'add_twig_toolkit' ) );
```

Comment:
**nitpick:** Remove the `add_twig_toolkit` hook

The `studiometa/ui` Twig extension extends the `studiometa/twig-toolkit` one, so we do not need to register it here anymore.

See https://github.com/studiometa/ui/blob/develop/packages/twig-extension/Extension.php#L19